### DISCUSSION TITLE
Event API

### DISCUSSION THREAD

**Discussion by chulinguy** [plausible/analytics#2387] [Anything Else]
I transferred ownership of our app to my boss today so he would automatically receive payment invoices.  
After the transfer, he also set up new payment information.  

However, now I am seeing a lot of `POST https://plausible.io/api/event net::ERR_BLOCKED_BY_CLIENT` console errors on our app.  We use the `plausible-tracker` npm on our React app.  No other changes to the app itself happened today.  

Please help.

**Comment by metmarkosaric** [comment]
hi @chulinguy! for these types of questions it's best to [contact our customer support](https://plausible.io/contact) as you can then share all the details on your account and we can easier see what's going on

**Comment by sandstrom** [comment]
It's likely blocked by an adblocker.

The underlying problem is that you cannot customize the path. See this issue (and vote for it) if you want that to change:
https://github.com/plausible/analytics/discussions/2414
